index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Personal Portfolio</title>
+  
+  <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+  <header>
+    <nav>
+      <ul class="container">
+        <li class="item"><a href="index.html">Welcome</a></li>
+        <li class="item"><img src="Assets\Images\hamburguer-menu.png"></li>
```

Comment:
- [ ] Kindly put the hamburger only in the nav tag

Reply:
I didnt understand this one. Can you explain in other words?